Autopilot/AttitudeManager/Src/attitudeStateClasses.cpp

```diff
@@ -131,7 +131,7 @@ void sendToSafetyMode::execute(attitudeManager* attitudeMgr)
 {
     SendToSafety_error_t ErrorStruct;
     float *channelOut = OutputMixingMode::GetChannelOut();
-    for(int channel = 0; channel < 4; channel++)
+    for(int channel = 0; channel < 8; channel++) //we use up to 7
```

Comment:
I feel like the comment may be misinterpreted by a future goosling. Can you write "We only use channels 0 to 7" instead please?